content/docs/1_guide/5_templates/8_escaping/guide.txt

```diff
@@ -0,0 +1,154 @@
+Title: Escaping content
+
+----
+
+Intro: Some content you output in your templates or snippets may be untrusted. For example it may come from user-generated content like a registration or comment form or you might use content from external sources like an API or external database. When you cannot fully trust the content, it is important to escape it correctly before it's sent to the browser to prevent cross-site scripting (XSS) attacks.
+
+----
+
+Text:
+
+## What is an XSS attack?
+
+(link: https://en.wikipedia.org/wiki/Cross-site_scripting text: Cross-site scripting) (XSS) is a type of security vulnerability that allows attackers to run arbitrary JavaScript in the browser of other, legitimate users of your site. This attack can be used to extract critical authentication information like session cookies from your users' browsers or to manipulate your site in unexpected ways.
```

Comment:
browsers of other, legitimate

Reply:
Ha! See, I can't spot my own typos either. ;)